nginx/version.txt

```diff
@@ -1 +1 @@
-0.49.0-1
+0.49.3
```

Comment:
why is it .3 instead of .1 or .2?

Reply:
Because we're using k8s.gcr.io/ingress-nginx/controller:v0.49.3

Reply:
ahh to match versions i s